docs/man-pages/condor_dagman.rst

```diff
@@ -1,297 +1,242 @@
+**condor_dagman** [**-help** | **-version**]
+
+**condor_dagman** **-p 0** **-f** **-l .** [*OPTIONS*] *DAG_file* [*DAG_file*...]
+
+**condor_dagman** **-p 0** **-f** **-l .** [**-WaitForDebug**]
+[**-CsdVersion** *VersionString*] [**-AllowVersionMismatch**]
+[**-Batch-Name** *name*] [**-Dag** *DAG_file*] [**-Debug** *level*]
+[**-Dot**] [**-DryRun**] [**-DumpRescue**] [**-Lockfile** *filename*]
+[**-Priority** *priority*] [**-SubmitMethod** *value*]
+[**-UseDagDir**] [**-AlwaysRunPost** | **-DontAlwaysRunPost**]
+[**-suppress_notification** | **-dont_suppress_notification**]
+[**-AutoRescue** *<0|1>*] [**-DoRescueFrom** *N*] [**-DoRecovery**]
+[**-load_save** *filename*] [**-MaxHold** *N*] [**-MaxIdle** *N*]
+[**-MaxJobs** *N*] [**-MaxPost** *N*] [**-MaxPre** *N*]
 
 Description
 -----------
 
-*condor_dagman* is a meta scheduler for the HTCondor jobs within a DAG
-(directed acyclic graph) (or multiple DAGs). In typical usage, a
-submitter of jobs that are organized into a DAG submits the DAG using
-*condor_submit_dag*. *condor_submit_dag* does error checking on
-aspects of the DAG and then submits *condor_dagman* as an HTCondor job.
-*condor_dagman* uses log files to coordinate the further submission of
-the jobs within the DAG.
-
-All command line arguments to the *DaemonCore* library functions work
-for *condor_dagman*. When invoked from the command line,
-*condor_dagman* requires the arguments *-f -l .* to appear first on the
-command line, to be processed by *DaemonCore*. The **csdversion** must
-also be specified; at start up, *condor_dagman* checks for a version
-mismatch with the *condor_submit_dag* version in this argument. The
-*-t* argument must also be present for the **-help** option, such that
-output is sent to the terminal.
-
-Arguments to *condor_dagman* are either automatically set by
-*condor_submit_dag* or they are specified as command-line arguments to
-*condor_submit_dag* and passed on to *condor_dagman*. The method by
-which the arguments are set is given in their description below.
-
-*condor_dagman* can run multiple, independent DAGs. This is done by
-specifying multiple **-dag** *a* rguments. Pass multiple DAG input
-files as command-line arguments to *condor_submit_dag*.
-
-Debugging output may be obtained by using the **-debug** *level*
-option. Level values and what they produce is described as
-
--  level = 0; never produce output, except for usage info
--  level = 1; very quiet, output severe errors
--  level = 2; normal output, errors and warnings
--  level = 3; output errors, as well as all warnings
--  level = 4; internal debugging output
--  level = 5; internal debugging output; outer loop debugging
--  level = 6; internal debugging output; inner loop debugging; output
-   DAG input file lines as they are parsed
--  level = 7; internal debugging output; rarely used; output DAG input
-   file lines as they are parsed
+A meta node scheduler that automatically manages HTCondor user workflows
```

Comment:
I'm not sure we want to use the term "meta scheduler" in dagman, but I'll leave that to you.